Sources/AudioKit/Nodes/Node+disconnectStrategy.swift

```diff
@@ -0,0 +1,66 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
+
+import AVFAudio
+
+/// Describes a way to disconnect a node from another node
+public enum DisconnectStrategy {
+    /// Recursively traverse node chain and disconnect
+    /// This strategy will keep connections from downstream nodes
+    /// to nodes that are not in current disconnect path
+    /// For example:
+    /// Mixer1
+    ///          ---> Node1
+    /// Mixer2
+    /// When disconnecting Node1 from Mixer1, Mixer2's connection to Node1
+    /// will be preserved
+    case recursive
+    /// Recursively traverse node chain and deatach nodes
```

Comment:
Sometimes misspelled in comments which escapes renaming refactoring so check for the word anywhere.

Reply:
Done.